3rdpartylibrary/google-maps-android-cmgn/src/main/java/com/google/maps/android/kml/KmlParser.java

```diff
@@ -66,17 +66,19 @@ public KmlParser(XmlPullParser parser) {
 
     /**
      * Parses the KML file and stores the created KmlStyle and KmlPlacemark
+     * @param documentBase Path leading to kml file directory to generate full path file url for map service.
+     *                     For example if the file was in C:/Documents/MyKMZ.kmz/test.kml this value would be C:/Documents/MyKMZ.kmz
      */
     /* package */
-    public void parseKml() throws XmlPullParserException, IOException {
+    public void parseKml(String documentBase) throws XmlPullParserException, IOException {
```

Comment:
I forgot to mention this before but please add the word final before all variables (for this part it should be in front of the parameters)